### PR TITLE
Add the SSL support

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

File: mockintosh/servers.py [hunk 1]
```diff
@@ -39,15 +42,32 @@ def load(self):
...
             rules = []
+            ssl = False
+            for service in services:
+                if ssl:
+                    break
+                ssl = service.get('ssl', False)
+
+            protocol = 'https' if ssl else 'http'
+            ssl_options = {
+                "certfile": path.join(__location__, 'ssl/cert.pem'),
```
Comment (undera) [inline mockintosh/servers.py:53]: If you use join, use it for all components, like `path.join(__location__, 'ssl', 'cert.pem'),`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cert:
+	openssl req \
+		-new \
+		-newkey rsa:4096 \
+		-days 3650 \
+		-nodes \
+		-x509 \
+		-subj "/C=US/ST=Denial/L=Springfield/O=Dis/CN=www.example.com" \
```
Comment (undera) [inline Makefile:8]: Correct settings are: Country: US, State: California, City: San Francisco, Organization: UP9.com, Domain: Mockintosh

File: mockintosh/servers.py (hunk 1 above)
Comment (mertyildiran) [reply mockintosh/servers.py]: @undera fixed with https://github.com/up9inc/mockintosh/commit/34b0c0c21a4b0e97ad48b4663317ff4d28cce224

File: Makefile (hunk 2 above)
Comment (mertyildiran) [reply Makefile]: @undera fixed with https://github.com/up9inc/mockintosh/commit/34b0c0c21a4b0e97ad48b4663317ff4d28cce224

**Pull request merged**